src/reports/components/report-sections/collapsible-script-provider.tsx

```diff
@@ -1,17 +1,23 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 export const addEventListenerForCollapsibleSection = (doc: Document) => {
-    const collapsibles = doc.getElementsByClassName('collapsible-control');
+    const collapsibles = doc.getElementsByClassName('collapsible-container');
 
     for (let index = 0; index < collapsibles.length; index++) {
         const self = collapsibles.item(index);
         self.addEventListener('click', () => {
-            const content = self.parentElement.nextElementSibling as HTMLElement;
+            const button = self.querySelector('.collapsible-control');
+            const content = button.parentElement.nextElementSibling as HTMLElement;
+            const expanded = button.getAttribute('aria-expanded') === 'false' ? false : true;
```

Comment:
I found some of the use of `expanded` a little tricky to follow, since this code is around toggling the whether the section is expanded. I wonder if it would be clearer to have `expanded` be `expandedBefore` and have `expandedAfter` set to `!expandedBefore`. Then we can set the `aria-expanded` attribute below to `expandedAfter`, set `aria-hidden` to `!expandedAfter`, and remove the `collapsed` class if `expandedAfter`. Together these changes might help the code read more naturally.

Reply:
Agreed; I've updated it. let me know what you think.

Reply:
Looks good! I like your choice of `wasExpandedBefore` and `isExpandedAfter`.